### PR TITLE
2213 allows for non-invertible transforms in TTA

### LINKED ISSUE #2213: Random Intensity transforms for test-time augmentation
Hi,

Test-time augmentation is a helpful tool to use but it seems that currently only spatial transformations are included for TTA. Intensity transformations such as Random contrast are not invertible transforms and thus cannot be used in current TTA implementation. Is there a way that I can use intensity transforms as TTA? Thanks.

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

Fixes #2213

### Description
this is a non-breaking change, removes the strong assumption of all invertibles:
https://github.com/Project-MONAI/MONAI/blob/8c0ff376587797aabd2abbdb569fb691880fc094/monai/data/test_time_augmentation.py#L136-L137

### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New tests added to cover the changes.
- [ ] Integration tests passed locally by running `./runtests.sh -f -u --net --coverage`.
- [ ] Quick tests passed locally by running `./runtests.sh --quick --unittests`.
- [ ] In-line docstrings updated.
- [ ] Documentation updated, tested `make html` command in the `docs/` folder.

### CODE REVIEW

File: monai/data/test_time_augmentation.py [hunk 1]
```diff
@@ -180,8 +185,10 @@ def __call__(
                 batch_output = batch_output.detach().cpu()
             if isinstance(batch_output, np.ndarray):
                 batch_output = torch.Tensor(batch_output)
-
-            transform_info = batch_data[transform_key]
+            transform_info = batch_data.get(transform_key, None)
+            if transform_info is None:
+                # no invertible transforms, adding dummy info for identity invertible
+                transform_info = [[0.0] for _ in range(self.batch_size)]
```
Comment (Nic-Ma) [inline monai/data/test_time_augmentation.py:191]: I am not sure whether we should add dummy data here, maybe just leave it to `None` if no invertible transforms?

Thanks.

**Review by Nic-Ma — APPROVED** [review]

File: monai/data/test_time_augmentation.py (hunk 1 above)
Comment (wyli) [reply monai/data/test_time_augmentation.py:191]: `None` doesn't work with one of the collate method

Comment (Nic-Ma) [reply monai/data/test_time_augmentation.py:191]: Ah, yes, that's the issue.
Or maybe we can use `"None"` to be consistent with other similar places, like:
https://github.com/Project-MONAI/MONAI/blob/dev/monai/transforms/spatial/dictionary.py#L259

Thanks.

**Pull request merged**